ntbs-service/Services/NotificationService.cs

```diff
@@ -242,11 +242,17 @@ private static void UpdateSocialRiskFactorsFlags(SocialRiskFactors socialRiskFac
 
         private static void UpdateRiskFactorFlags(RiskFactorDetails riskFactor)
         {
-            if (riskFactor.Status != Status.Yes)
+            if (riskFactor.Status == Status.Yes)
             {
-                riskFactor.IsCurrent = false;
-                riskFactor.InPastFiveYears = false;
-                riskFactor.MoreThanFiveYearsAgo = false;
+                riskFactor.IsCurrent = riskFactor.IsCurrent ?? false;
```

Comment:
should this stay null if we have it nullable? Or should it be false when user marks risk factor as yes?

Reply:
If they have marked it as yes then they have been shown the checkboxes, and chosen not to tick them - which is safe to consider as a false.